api/src/org/apache/cloudstack/api/command/user/vpc/RestartVPCCmd.java

```diff
@@ -66,14 +66,14 @@ public Boolean getCleanup() {
         if (cleanup != null) {
             return cleanup;
         }
-        return true;
+        return false;
```

Comment:
The same is valid here

Reply:
@rafaelweingartner when the option is not passed, the value should be false which is what the fix it.

Reply:
This means that the default is `false`, right?
Can't you declare the variable as `private boolean cleanup = false`?